lib/helper/WebDriverIO.js

```diff
@@ -924,7 +1002,71 @@ class WebDriverIO extends Helper {
       }).then((selected) => {
         if (!Array.isArray(selected)) selected = [selected];
         selected = selected.filter((val) => val === true);
-        return assert.equal(selected.length, num, `expected nummber of visible elements (${locator}) is ${num}, but found ${selected.length}`);
+        return assert.equal(selected.length, num,
+          `expected number of visible elements (${locator}) is ${num}, but found ${selected.length}`);
+      });
+    });
+  }
+
+  /**
+   * Checks that there is an Element with given CSS property.
+   */
+  seeCssPropertyOnElement(locator, cssProperty, value) {
+    let client = this.browser;
+    return this._locate(locator).then((res) => {
+      if (!res.value || res.value.length === 0) {
+        throw new Error(`${locator} cant be located by name|text|CSS|XPath`);
+      }
+      let commands = [];
+      res.value.forEach((el) => commands.push(client.elementIdCssProperty(el.ELEMENT, cssProperty)));
+      return client.unify(commands, {
+        extractValue: true
+      }).then((selected) => {
+        if (!Array.isArray(selected)) selected = [selected];
+        selected = selected.filter((val) => val === value);
+        return assert.ok(selected.length > 0,
+          `Element ${locator} hasn't CSS property ${cssProperty} with value ${value}`);
+      });
+    });
+  }
+
+  /**
+   * Checks that there is an Element with given attribute.
+   */
+  seeAttributeOnElement(locator, attr, value) {
```

Comment:
I don't think that mixing this is good idea. This will lead to large methods which in the future will be more difficult to maintain